jsonx/internal/gjson_test.go

```diff
@@ -23,10 +24,9 @@ func TestRandomData(t *testing.T) {
 			panic(v)
 		}
 	}()
-	rand.Seed(time.Now().UnixNano())
 	b := make([]byte, 200)
 	for i := 0; i < 2000000; i++ {
-		n, err := rand.Read(b[:rand.Int()%len(b)])
+		n, err := cryptorand.Read(b[:rand.Int()%len(b)])
```

Comment:
why is it better to use crypto rand? (given math.Rand is now seeded randomly at startup)

Reply:
The linter says rand.Read is deprecated as of go 1.20... The other option is to rewrite to do something different or ignore the warning, and this seemed like the best.